robottelo/ui/user.py

```diff
@@ -176,8 +181,36 @@ def admin_role_to_user(self, username):
         admin_role_locator = locators['users.admin_role']
         is_admin_role_selected = self.find_element(
             admin_role_locator).is_selected()
-        if not is_admin_role_selected:
+        if is_admin_role_selected != assign:
             self.click(admin_role_locator)
             self.click(common_locators['submit'])
-            is_admin_role_selected = True
+            is_admin_role_selected = assign
         return is_admin_role_selected
+
+    def validate_user(
+            self, username, field_name, field_value, validate_table=True):
+        """Checks if selected user has necessary field values on the index and
+        edit pages
+        """
+        element = self.search(username)
+        if element is None:
+            raise UINoSuchElementError(
+                'Unable to find the username "{0}".'.format(username)
+            )
+        if validate_table:
```

Comment:
Can we change the `validate_table` to `validate_index_page` ? just a thought. 
I'm assuming we are validating the user fields from user's index page that lists are users. ?